package.json

```diff
@@ -1,25 +1,28 @@
 {
-	"name": "whatsapp-chatgpt",
```

Comment:
Please run formatter once,

npm run prettier

Reply:
Done. Thank you!